shared/src/test/diff/fcp/Overloads.mls

```diff
@@ -100,47 +100,32 @@ def f = fun x -> (if true then IISS else BBNN) x
 //│     [+'a, -'b] in {[bool, bool], [number, number]}
 //│     [+'a, -'b] in {[int, int], [string, string]}
 
-f: int -> number
-//│ res: int -> number
-
-// FIXME
 f(0)
-//│ res: 'a
-//│   where
-//│     'a :> number
-//│     [+'b, -'a] in {[bool, bool], [number, number]}
-//│     [+'b, -'a] in {[int, int], [string, string]}
+//│ res: number
 
 // FIXME
```

Comment:
It seems the error is correct (`+` takes integers, not numbers), but the type provenance locations are all wrong ("flows into integer literal").